warehouse/table_upload.go

```diff
@@ -47,12 +47,7 @@ func createTableUploads(uploadID int64, tableNames []string) (err error) {
 			valueRefsArr = append(valueRefsArr, fmt.Sprintf(`$%d`, index))
 		}
 		valueReferences = append(valueReferences, fmt.Sprintf("(%s)", strings.Join(valueRefsArr, ",")))
-		valueArgs = append(valueArgs, uploadID)
-		valueArgs = append(valueArgs, tName)
-		valueArgs = append(valueArgs, "waiting")
-		valueArgs = append(valueArgs, "{}")
-		valueArgs = append(valueArgs, currentTime)
-		valueArgs = append(valueArgs, currentTime)
+		valueArgs = append(valueArgs, uploadID, tName, "waiting", "{}", currentTime, currentTime)
```

Comment:
This won't really improve our performance, because we are allocating capacity for this slice in advance. So lets not make this change and add a skipcq rule for this.

Reply:
Initially thought the same.
But as per skipcq rules : https://deepsource.io/docs/how-to/silence-issues.html
It has to be added to every line to suppress this. Hence made the change just to prevent from adding extra comments.